mu-plugins/blocks/global-header-footer/header.php

```diff
@@ -9,16 +9,32 @@
 /**
  * Defined in `render_global_header()`.
  *
+ * @var array  $attributes
  * @var array  $menu_items
  * @var string $locale_title
  * @var string $show_search
  */
 
-$container_class = 'global-header';
+$container_class = 'global-header has-text-color has-background';
```

Comment:
Is it worth denoting which styles are Gutenberg for future developers? I think `has-text-color` and `has-background` are correct?

Reply:
I'm not sure what you mean - I'm mirroring the way colors are applied to group blocks with `has-X-color` and `has-X-background-color`, so I've added these two for consistency. The only thing I didn't add was the `backgroundColor` & `textColor` attributes to the group comment, since I don't think they're necessary.

Reply:
I don't see these styles in any associated CSS styles so my assumption is that these CSS classes are coming from Gutenberg. We may be assuming that future developers are familiar with Gutenberg styling conventions (and business logic).